### ISSUE TITLE
Create a youtube channel for the meeting video recordings

### ISSUE THREAD

**Issue by echarles** [jupyterlab/rtc#60]
Would be useful so folks can easily find the recordings.

**Comment by saulshanabrook** [comment]
They are on the Jupyter youtube channel at the moment: https://www.youtube.com/watch?v=fwvH53GXxM8&feature=youtu.be

I have just been bad at remembering to record and upload meetings.

**Comment by echarles** [comment]
Is that a [YouTube Channel](https://support.google.com/youtube/answer/1646861?hl=en)? The url should be something like `/channel/...` Example: https://www.youtube.com/channel/UCdkGV51Nu0unDNT58bHt9bg

There is also the playlist option: https://www.youtube.com/playlist?list=PLvmPtYZtoXOENHJiAQc6HmV2jmuexKfrJ

**Comment by saulshanabrook** [comment]
Yeah it is a playlist on the Jupyter channel: https://www.youtube.com/playlist?list=PLUrHeD2K9Cmk5PpU7a3Pf5zEteJ-_kA81
